components/composer/src/lib/utils.ts

```diff
@@ -16,6 +16,24 @@ export const debounce = (
     timeout = setTimeout(later, wait);
   };
 };
+
 export const isValidEmail = (email: string): boolean => {
   return /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(email);
 };
+
+export const calculateMessageTempSnippet = (body: string): string => {
```

Comment:
```suggestion
export const cleanMessageSnippet = (body: string): string => {
```

Not sure if we're doing a calculation here.

Reply:
@AaronDDM I basically just copied the similar name from the function they used for calculating the snippet in cloud-core. But yeah, cleanMessage makes more sense.